src/iotconnect_telemetry.c

```diff
@@ -63,39 +71,57 @@ cJSON *json_object_dotset_locate(cJSON *search_object, char **leaf_name, const c
     free(mutable_path);
     return target_object;
 
-    cleanup:
+cleanup:
     free(mutable_path);
     return NULL;
 }
 
 static cJSON *setup_telemetry_object(IotclMessageHandle message) {
     IotclConfig *config = iotcl_get_config();
-    if (!config) return NULL;
-    if (!message) return NULL;
+    if (!config) {
+        return NULL;
+    }
+    if (!message) {
+        return NULL;
+    }
 
     cJSON *telemetry_object = cJSON_CreateObject();
-    if (!telemetry_object) return NULL;
-    if (!cJSON_AddStringToObject(telemetry_object, "id", config->device.duid)) goto cleanup_to;
-    if (!cJSON_AddStringToObject(telemetry_object, "tg", "")) goto cleanup_to;
+    if (!telemetry_object) {
+        return NULL;
+    }
+    if (!cJSON_AddStringToObject(telemetry_object, "id", config->device.duid)) {
```

Comment:
The purpose of the inline style was to improve readibility by showing in fewest lines as possible what is actually going on. While not correct, it was an exception that I decided to make in order to make the tradeoff. Now you decided to unmake that decission, so we need to discuss this, but I am even sure how we can conclude the discussion without arbitration, as this is subjective.